validators/phone_IN.js

```diff
@@ -0,0 +1,17 @@
+const dry = require('drytypes')
+
+module.exports = dry.makeDryType((x) => {
+
+  if (typeof x !== 'string') {
+    return { success: false, message: 'Phone number should be string!' }
+}
+// const phoneRegex = /^(\+?91|0)?[6789]\d{9}$/;
+const phoneRegex = /(((^[\+,0][9][1])(((\s[0-9]{7,10})|(\S[0-9]{7,10}))|([-]\S[0-9]{7,10})))|((^[\+,0][2]{2,2})((\S[0-9]{7,8})|((([-])[0-9]{7,8})|(\s[0-9]{7,8})))))|(((^[6,7,8,9][0-9]{9,9}))|(^[0,\+](([9][1)|[6,7,8,9]))[0-9]{8,9}))/gm;
```

Comment:
I wonder if we can manage to validate all countries' phone numbers withing single validator lib

Reply:
hmm that's a good idea